controller/utils.go

```diff
@@ -1144,9 +1129,12 @@ func AcquireLock(
 			l.Warn("timeout while wating for redis lock")
 			return nil, false, errors.New("timeout while wating for redis lock")
 		case <-ticker.C:
-			if operationManager != nil && operationManager.WasCanceled() {
-				l.Warn("operation was canceled")
-				return nil, true, nil
+			if operationManager != nil {
```

Comment:
shouldn't you use inRollingUpdate here as well?

Reply:
AcquireLock is used in other places as well and I didn't want to change every caller of this function.